workspaces/liferay-learn-workspace/client-extensions/liferay-learn-custom-element/src/utils/constants.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+export const endpoint = {
+	learningPath: '/o/c/learningpaths',
+
+	modules: '/o/c/modules/',
```

Comment:
you need to decide if they are plural or singular

I'd recommend to use the constant name as endpoints, and then all endpoints in a singular form